tasks/users.yml

```diff
@@ -53,3 +53,5 @@
     owner: root
     group: shadow
     state: present
+  tags:
+    - root_user
```

Comment:
Why do you need a tag specifically for the root user? Do you not want to execute this task?

Reply:
Indeed, it should be possible to keep the root user active, maybe it's nicer to add a var named deny_root_login and check for it's value?

Reply:
No, I'm against this, why would we need the root user?

Reply:
It's the default to remove the root user, but there should be an option to keep them. The default is and should be to disable root logins.

Reply:
The whole security effict oft this task is gone if we allow to disable this. On what machine do you need a root user?